app/Models/Profile.php

```diff
@@ -51,21 +50,18 @@ public function user(): BelongsTo
 
     public function getAvatar(): string
     {
-        return $this->getAvatarGenerator()
-            ->backgroundColor(ColorGenerator::generate($this->full_name))
-            ->image();
+        $avatar = new UiAvatar();
+        $avatar->background = ColorGenerator::generate($this->full_name);
+        $avatar->name = mb_substr($this->first_name, 0, 1) . mb_substr($this->last_name, 0, 1);
```

Comment:
What do you think about to use `charAt()` method instead of `mb_substr()`?

https://laravel.com/docs/11.x/strings#method-char-at

Reply:
I don't see any advantage. `charAt()` looks nice if we don't know if there's a character under some index. Here we are sure that every name and surname has first letter.